### DISCUSSION TITLE
How to Link Account of different sellers after Etsy Commercal Acsess

### DISCUSSION THREAD

**Discussion by TradeportApi** [etsy/open-api#1155] [Q&A] [answered]
Hello,

I prepared an app and bought Commercal Acsess. I want to offer my product to different sellers, but I don't know how to connect their accounts through the website and integration panel I prepared :(

Does anyone have experience in this matter?

Sincerely regards,

**Comment by Gareth064** [comment, marked as answer]
Hey

No idea what language you are working in, but this Etsy tutorial is a good place to start

https://developers.etsy.com/documentation/tutorials/quickstart#create-the-client-landing-page

I would also read through the Authorization documentation very carefully

https://developers.etsy.com/documentation/essentials/authentication

But in short

1. Build a URL with all the relevant Auth requirements,
2. Send your user to that URL
3. They log into Etsy and approve your apps access
4. Etsy will redirect them to somewhere else on your app, and provide an auth code
5. You capture that auth code and use it to get an access token (plus refresh token) for that user.
6. Persist the tokens
7. Use the tokens.

**Reply by TradeportApi** [reply]
Hello,

Thanks for your reply and support. I want ask another question do my clients(sellers) have to get personal access  from Etsy Api app page? Or that link you mentioned is enough for my link?

Thanks,

**Reply by Gareth064** [reply]
Hey. Yes the link will handle all the authorization. Clients/Sellers don't have to do anything except give your App permission to their Shop (when it asks). You then have to manage tokens to make sure you continue to have access.

Two other things to know.
1. Clients/Sellers can revoke your apps access at any time from within their shop.
2. If the Client/Seller changes their password to their shop, this can sometimes cause all apps to request permissions again.
